timeseries/src/autogluon/timeseries/models/chronos/model.py

```diff
@@ -216,13 +216,6 @@ def __init__(
             )
         self.context_length = hyperparameters.get("context_length")
 
-        if self.context_length is not None and self.context_length > self.maximum_context_length:
-            logger.info(
-                f"\tContext length {self.context_length} exceeds maximum context length {self.maximum_context_length}."
-                f"Context length will be set to {self.maximum_context_length}."
-            )
-            self.context_length = self.maximum_context_length
-
         # we truncate the name to avoid long path errors on Windows
```

Comment:
Should we add some unit tests to make sure that non of the other hyperparameters (excluding `model_path`) break HPO?